x-pack/plugins/observability_solution/observability/server/lib/rules/custom_threshold/custom_threshold_executor.test.ts

```diff
@@ -2223,6 +2223,1692 @@ describe('The custom threshold alert type', () => {
   });
 });
 
+describe('The custom threshold rule type with last_value aggregation', () => {
```

Comment:
nit: can you please wrap both describes in one overall describe? It will help run all the tests together when we change something in the executor.

Reply:
added overall describe block in c1fa8f42ba5a2db243943b9988f8d1795ca8ae3a. I thought all tests should run every time unless we skip them. Is there a possibility where tests would not run together if they are in separate describe blocks? Could you give an example?

Reply:
Ah, I meant to run it locally, not in CI. Locally, when I change something in the executor, I run the first describe section in the related test in IDE to make sure everything is fine. So the nit was more for convenience.